contracts/0.8.25/vaults/StakingVault.sol

```diff
@@ -349,46 +346,59 @@ contract StakingVault is IStakingVault, Ownable2StepUpgradeable {
         address _refundRecipient
     ) external payable onlyOwner {
         if (msg.value == 0) revert ZeroArgument("msg.value");
-        if (_pubkeys.length == 0) revert ZeroArgument("_pubkeys");
         if (_amounts.length == 0) revert ZeroArgument("_amounts");
+        if (_pubkeys.length == 0) revert ZeroArgument("_pubkeys");
         if (_pubkeys.length % PUBLIC_KEY_LENGTH != 0) revert InvalidPubkeysLength();
+        if (_pubkeys.length / PUBLIC_KEY_LENGTH != _amounts.length) revert PubkeyLengthDoesNotMatchAmountLength();
 
         // If the refund recipient is not set, use the sender as the refund recipient
-        if (_refundRecipient == address(0)) {
-            _refundRecipient = msg.sender;
-        }
+        if (_refundRecipient == address(0)) _refundRecipient = msg.sender;
 
         uint256 feePerRequest = TriggerableWithdrawals.getWithdrawalRequestFee();
         uint256 totalFee = (_pubkeys.length / PUBLIC_KEY_LENGTH) * feePerRequest;
         if (msg.value < totalFee) revert InsufficientValidatorWithdrawalFee(msg.value, totalFee);
 
         TriggerableWithdrawals.addWithdrawalRequests(_pubkeys, _amounts, feePerRequest);
 
-        uint256 excess = _refundExcess(totalFee, _refundRecipient);
+        uint256 excess = msg.value - totalFee;
+        if (excess > 0) {
+            (bool success, ) = _refundRecipient.call{value: excess}("");
+            if (!success) revert TransferFailed(_refundRecipient, excess);
+        }
 
         emit ValidatorWithdrawalsTriggered(msg.sender, _refundRecipient, _pubkeys, _amounts, excess);
     }
 
-    function triggerValidatorExits(bytes calldata _pubkeys, address _refundRecipient) external payable {
+    /**
+     * @notice Triggers EIP-7002 validator exits by the node operator.
+     *         Because the node operator cannot ensure that all the associated validators are under control,
+     *         the node operator has the ability to forcefully eject validators.
+     * @param _pubkeys Concatenated validators public keys, each 48 bytes long
+     * @param _refundRecipient Address to receive the fee refund, if zero, refunds go to msg.sender
+     * @dev    The caller must provide sufficient fee via msg.value to cover the withdrawal request costs
```

Comment:
Perhaps in this section (and in the previous function's docs), we must mention to users using the `calculateValidatorWithdrawalFee` function to calculate those costs?